example/cavity/Fluid/constant/dynamicMeshDict

```diff
@@ -0,0 +1,33 @@
+/*--------------------------------*- C++ -*----------------------------------*\
+| =========                 |                                                 |
+| \\      /  F ield         | OpenFOAM: The Open Source CFD Toolbox           |
+|  \\    /   O peration     | Version:  v1912                                 |
+|   \\  /    A nd           | Website:  www.openfoam.com                      |
+|    \\/     M anipulation  |                                                 |
+\*---------------------------------------------------------------------------*/
```

Comment:
Preferably remove these headers and the respective comment rulers below. These files are not part of OpenFOAM and we try to support different OpenFOAM versions at the same time.